backend/windmill-worker/src/worker.rs

```diff
@@ -1385,17 +1387,17 @@ pub async fn run_worker<R: rsmq_async::RsmqConnection + Send + Sync + Clone + 's
 
         if (jobs_executed as u32 + vacuum_shift) % VACUUM_PERIOD == 0 {
             let db2 = db.clone();
-            let worker_name2 = worker_name.clone();
-            tokio::task::spawn(async move {
-                tracing::info!(worker = %worker_name2, "vacuuming queue and completed_job");
+            let current_span = tracing::Span::current();
+            tokio::task::spawn((async move {
+                tracing::info!("vacuuming queue and completed_job");
                 if let Err(e) = sqlx::query!("VACUUM (skip_locked) queue")
                     .execute(&db2)
                     .await
                 {
-                    tracing::error!(worker = %worker_name2, "failed to vacuum queue: {}", e);
+                    tracing::error!("failed to vacuum queue: {}", e);
                 }
-                tracing::info!(worker = %worker_name2, "vacuumed queue and completed_job");
-            });
+                tracing::info!("vacuumed queue and completed_job");
+            }).instrument(current_span));
```

Comment:
Please confirm if it makes sense to move the span context here. If not I'll remove it